isis/src/qisis/objs/StretchTool/AdvancedStretch.cpp

```diff
@@ -105,6 +118,41 @@ namespace Isis {
   }
 
 
+  /**
+   * Used to restore a saved Stretch from a cube. This function is 
+   * distinct from setStretch in that setStretch delibrately _does not_ 
+   * change the stretch type, and this function does change the stretch type.  
+   *  
+   * @param newStretch saved stretch to restore
+   */
+  void AdvancedStretch::setStretchFromCube(Stretch newStretch) {
+    QString stretchTypeName = newStretch.getType(); 
+
+    int index = 0;
+    if (stretchTypeName.compare("LinearStretch") == 0 ) {
+      index = 0; 
+    }
+    else if (stretchTypeName.compare("SawtoothStretch") == 0 ) {
+      index = 1; 
+    }
+    else if (stretchTypeName.compare("BinaryStretch") == 0) {
+      index = 2; 
+    }
+    else if (stretchTypeName.compare("ManualStretch") == 0) {
+      index = 3;
+    }
+    // Fail by defaulting to Linear
+
+
+//    p_stretchTypeStack->setCurrentIndex(index); <- does not work. 
```

Comment:
Why is this comment here?

Reply:
It was originally so that I would remember that it didn't work, but the codebase doesn't need this.